### ISSUE TITLE
Syntax highlighting contrast fails Web Content Accessibility Guidelines

### ISSUE THREAD

**Issue by dadlerj** [sourcegraph/sourcegraph#14546]
Reported by https://app.hubspot.com/contacts/2762526/company/693777200

Discussion in Slack: https://sourcegraph.slack.com/archives/C0HMGV90V/p1602173122127700

>What customization/color options do we have?  I know there are some branding options, but have received feedback about colors that was unexpected.

And from the end user:

>Light theme, comments are `#93A1A1` on `#FFFFFF`, with a contrast of 2.67:1, which fails pretty much everything.
>
>Dark theme, they’re `#2B8A3E` on `#0E121B`, with a contrast of 4.28:1, which only passes WCAG AA for large text (and objects/UI components), but fails AAA and AA for normal text.

The user suggested that simply adding a high contrast option would be a fine workaround for now

**Comment by dadlerj** [comment]
Note from @slimsag :

>Worth reiterating that editing this is really pretty straightforward:
>
https://sourcegraph.slack.com/archives/C0W2E592M/p1596479729021400?thread_ts=1596324374.013800&cid=C0W2E592M
>
>It also wouldn't be hard for us to offer a high-contrast syntax highlighting theme option in the user settings. I would love that (although it wouldn't match the rest of the app)

**Comment by slimsag** [comment]
Merged https://github.com/slimsag/syntect/pull/7 and will go out in the next (not current) release. It will help with the comment colors only.

**Comment by felixfbecker** [comment]
Can this be closed then?

**Comment by slimsag** [comment]
> It will help with the comment colors only.

No, this issue isn't fully fixed I believe.

**Comment by jasongornall** [comment]
This is being attacked by @umpox in a large scale effort to fix contrast everywhere. Tom feel free to reopen if you think this should still exist.

**Comment by felixfbecker** [comment]
@jasongornall is there a tracking issue for that effort where this is mentioned/tracked? Just a bit worried that with closing all the individual issues in favor of the "large scale effort", some of these aspects might get forgotten about/are not verified at the end.

**Comment by umpox** [comment]
@felixfbecker

The work to fix this has been merged in https://github.com/sourcegraph/sourcegraph/pull/37385. It is currently feature flagged so we can test internally before enabling it site-wide.

We have an issue (https://github.com/sourcegraph/sourcegraph/issues/37535) to enable it fully over the coming weeks, we can continue further discussion there to avoid duplicating many issues.
